### PR TITLE
Updated Gnomonic Projection to include "central_longitude" - 'lon_0' Arg

### PR DESCRIPTION
**Pull request by cvelascof**

### CODE REVIEW

**Comment by pp-mo** [comment]
Hi there @cvelascof 
Looks fine to me really, you just need to fix (a) a pep8 failure and (b) a header date issue
In case you're not familiar with this stuff, here are the relevant messages from your Travis test log :
```
test_pep8_conformance (cartopy.tests.test_coding_standards.TestCodeFormat) ... /home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/Cartopy-0.15.0-py2.7-linux-x86_64.egg/cartopy/crs.py:1274:80: E501 line too long (80 > 79 characters)
/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/Cartopy-0.15.0-py2.7-linux-x86_64.egg/cartopy/crs.py:1275:80: E501 line too long (100 > 79 characters)
FAIL
test_future_imports (cartopy.tests.test_coding_standards.TestFutureImports) ... ok
test_license_headers (cartopy.tests.test_coding_standards.TestLicenseHeaders) ... The file header at lib/cartopy/crs.py is out of date. The last commit was in 2017, but the copyright states it was 2016.
ERROR
```
( The header date is right in the first line of the file :wink: )

I *would* also ask for a tiny test, but as we have no existing test for this class at all, I think we can just skip it for something this small (!)

**Comment by ajdawson** [comment]
@cvelascof - can you complete the CLA form at http://scitools.org.uk/documents/cla_form.pdf please. The form contains instructions about what it is and what to do with it.

**Comment by pp-mo** [comment]
Thanks @ajdawson, I forgot that !

File: lib/cartopy/crs.py [hunk 1]
```diff
@@ -1271,8 +1271,10 @@ def threshold(self):
 
 
 class Gnomonic(Projection):
-    def __init__(self, central_latitude=0.0, globe=None):
-        proj4_params = [('proj', 'gnom'), ('lat_0', central_latitude)]
+    def __init__(self, central_latitude=0.0,
```
Comment (ajdawson) [inline lib/cartopy/crs.py]: This line has some whitespace at the end, the last character on the line should be the `,`.

**Comment by cvelascof** [comment]
CLA completed and submitted to MetOffice

**Comment by pp-mo** [comment]
Thanks, welcome to Iris @cvelascof !

**Pull request merged**